firmware/src/usb_descriptors.c

```diff
@@ -101,7 +102,7 @@ static const uint8_t g_config_descriptor[] = {
   1,  // Index value of this configuration
   0,  // Configuration string index
   USB_ATTRIBUTE_DEFAULT | USB_ATTRIBUTE_SELF_POWERED,  // Attributes
-  50,  // Max power consumption (2X mA)
+  USB_POWER_CONSUMPTION,  // Max power consumption (2X mA)
```

Comment:
This seems to contradict the comment above. I think you want mA/2 for the value here.